xbmc/pvr/guilib/PVRGUIActions.cpp

```diff
@@ -1551,9 +1551,14 @@ namespace PVR
   {
     CFileItemPtr pvrItem(item);
     if (URIUtils::IsPVRChannel(item->GetPath()) && !item->HasPVRChannelInfoTag())
-      pvrItem = std::make_shared<CFileItem>(
+    {
+      const std::shared_ptr<PVR::CPVRChannelGroupMember> groupMember =
           CServiceBroker::GetPVRManager().ChannelGroups()->GetChannelGroupMemberByPath(
-              item->GetPath()));
+              item->GetPath());
+      if (!groupMember)
+        return false;
```

Comment:
No need for the early return.

if (groupMember)
 pvrItem =
 
 Nothing else needed.

Reply:
done